lib/manageiq/providers/ovirt/legacy/inventory.rb

```diff
@@ -166,13 +166,14 @@ def collect_primary_jobs(jobs)
 
           def collect_primary_targeted_jobs(jobs)
             results = collect_in_parallel(jobs) do |key, ems_ref|
+              xpath = '/host' if key == :host
```

Comment:
Normally we would prefer something like this to make it more obvious that the `nil` is intentional.
```ruby
xpath = key == :host ? '/host' : nil
```